sympy/stats/rv.py

```diff
@@ -11,7 +11,7 @@
 sympy.stats.frv
 sympy.stats.rv_interface
 """
-
+from sympy.core.compatibility import string_types
```

Comment:
It would be good to have this line after the next line, i.e. `from __future__ import ...` should come before this.